string[] sequence
int32[] time
int32 loop
---
string message
